Packaging.Targets/Rpm/RpmFileColor.cs

```diff
@@ -2,41 +2,21 @@
 
 namespace Packaging.Targets.Rpm
 {
+    /*
+     * from https://github.com/rpm-software-management/rpm/blob/master/build/rpmfc.h
+     */
+
     /// <summary>
     /// Determines the type of the file.
     /// </summary>
     [Flags]
-    internal enum RpmFileColor
+    internal enum RpmFileColor : int
     {
         RPMFC_BLACK = 0,
         RPMFC_ELF32 = 1 << 0,
         RPMFC_ELF64 = 1 << 1,
         RPMFC_ELFMIPSN32 = 1 << 2,
-        RPMFC_PKGCONFIG = 1 << 4,
```

Comment:
Can we keep these enum values?  They map to the values defined in the RPM API I would prefer to keep them in sync.

Reply:
I did some research before I pulled those values out. 

The values you have here got [dropped from RPM in 2010](https://github.com/rpm-software-management/rpm/commit/ad2c8485ea109d37a2fb8ee9ed5f688eef282bf8#diff-b31783559c8ac825c2a3fa3368e603b5) and RPM's classification model doesn't use them anymore but rather uses the text classifier dictionary. My edits align to the current [rpmfc.h header](https://github.com/rpm-software-management/rpm/blob/master/build/rpmfc.h).

Reply:
OK, sounds good then!